LogicClass.py

```diff
@@ -103,7 +114,13 @@ def setup(self, network):
 
```

Comment:
please put documentation blob
also (self, peer)
Nitpicky I know but spaces make stuff easier to read